From 719061882bab1d2b2a9f89d7c1295bcab0218cfb Mon Sep 17 00:00:00 2001
From: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Date: Mon, 6 Oct 2014 09:34:07 +0100
Subject: [PATCH 338/351] ASoC: florida: Reduce delay after SYSCLK patch is
 applied

The 10mS is significantly more than is required after updating the write
sequencer registers. This patch reduces the delay to 1mS.

Change-Id: I1482b82eda2ff0b9384706326256febcdc8db706
Signed-off-by: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
---
 sound/soc/codecs/florida.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/florida.c b/sound/soc/codecs/florida.c
index 9b068b7..2448320 100644
--- a/sound/soc/codecs/florida.c
+++ b/sound/soc/codecs/florida.c
@@ -215,7 +215,7 @@ static int florida_sysclk_ev(struct snd_soc_dapm_widget *w,
 		break;
 	}
 
-	msleep(10);
+	udelay(1000);
 
 	return 0;
 }
-- 
1.7.9.5

